exir/emit/test/test_emit.py

```diff
@@ -1838,8 +1838,40 @@ def forward(self, x):
         ep = to_edge(ep)
         # Lower the graph to executorch.
         ep = ep.to_executorch(
-            config=ExecutorchBackendConfig(emit_mutable_buffer_names=True)
+            config=ExecutorchBackendConfig(
```

Comment:
Maybe also add a __post_init__ in the `ExecutorchBackendConfig` dataclass that asserts if `emit_mutable_buffer_names` is `False` and `alloc_mutable_buffers` is also False.

Reply:
I considered this but since the MemoryPlanningPass is user configureable there is no guarantee memory_planning_pass.alloc_mutable_buffers exists